pkg/ui/chooser/embed/index.html

```diff
@@ -16,15 +17,35 @@
             grid-template-columns: 1fr 1fr;
             column-gap: 2rem;
         }
+
+        header {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            text-align: center;
+        }
+
+        .logo {
+            max-width: 100%;
+            height: 100%;
```

Comment:
let's go with `height: 150px;` to keep the logo quite impactful

Reply:
you'll probably need to go with something like `margin-top: -25px;` for `.logo` and `margin-top: -35px;` for `.title p` to have something similar to what we used to have